packages/oscal-react-library/src/components/OSCALCatalogManageGroup.tsx

```diff
@@ -0,0 +1,1052 @@
+      setShowCardMenu(false);
+    }
+    function handleIncreaseIndent() {
+      const sibling = selectedGroup.rightSibling;
+      if (sibling === undefined) {
+        setShowCardMenu(false);
+        return;
+      }
+      MoveGroup(selectedGroup.groupID, selectedGroup.groupTitle, sibling.groupID);
+      // Reload the main data
+      getData();
+      setShowCardMenu(false);
+    }
+    function handleDecreaseIndent() {
+      const parentID = selectedGroup.parentID;
+      if (parentID === undefined) {
+        setShowCardMenu(false);
+        return;
+      }
+
+      const parent = orderGroups.find((x) => x.groupID === parentID) ?? defaultGroup;
+      const new_parent_id = parent.parentID;
+
+      MoveGroup(selectedGroup.groupID, selectedGroup.groupTitle, new_parent_id);
+      // Reload the main data
+      getData();
+      setShowCardMenu(false);
+    }
+    function handleSaveNewGroup() {
+      if (edit && !addBelow) {
+        EditGroupTitle(selectedGroup.groupID, Name);
+        setAddNewGroup(false);
+        getData();
+        setEditGroup(defaultGroup);
+        return;
+      }
+      if (Name.length < 1) return;
+      if (preID === "") {
+        ID = Name.substring(0, 1) + ID;
+      }
+      if (addBelow) {
+        const parent_id = newGroupParent === undefined ? "" : newGroupParent.groupID;
+        SaveNewGroup(ID, Name, parent_id);
+        setAddNewGroup(false);
+        getData();
+        setAddBelow(false);
+        return;
+      }
+      SaveNewGroup(ID, Name, "");
+      setAddNewGroup(false);
+      getData();
+    }
+    function handleDeleteGroup() {
+      //setOpenDeleteDialog(true);
+      DeleteGroup(selectedGroup.groupID);
+      getData();
+      setAddNewGroup(false);
+    }
+
+    const isSelected = selectedItemName === "NewGroup" ? true : false;
+    return (
+      <Container
+        onClick={handleNewGroupClick}
+        sx={{ height: 73, border: "1px solid #D2D2D2", background: "#EAEAEA", width: 312 }}
+      >
+        <Container
+          sx={{
+            height: 73,
+            left: 20,
+            position: "absolute",
+            background: "#F6F6F6",
+            border: "1px solid #D2D2D2",
+            width: 300,
+          }}
+        >
+          <Grid sx={{ height: 73 }}>
+            <Grid sx={{ height: 40 }}>
+              <IconButton sx={{ left: 0, top: 2, position: "absolute" }}>
+                <DragIndicatorIcon />
+              </IconButton>
+              <TextField
+                size="small"
+                label="ID"
+                defaultValue={addBelow ? undefined : selectedItemGroup?.groupID.substring(0, 2)}
+                onChange={handleEditIDChange}
+                sx={{
+                  left: 32,
+                  height: 35,
+                  width: 52,
+                  top: 5,
+                  position: "absolute",
+                  fontSize: 10,
+                  fontWeight: 200,
+                  background: "#ffffff",
```

Comment:
This is `white` in the palette theming

Reply:
The same happened when I directly use a palette already defined 
<img width="778" alt="image" src="https://github.com/EasyDynamics/oscal-react-library/assets/59936247/c6f5d1f8-b6a3-4128-b3d7-80a6f745b1e5">
It looks like typescript needs to extra directive to accept certain palettes defined in AppTheme.js

Reply:
You're correct on this. You can follow the same method as I had mentioned with the `lightGray` palette option above.